rust/tools/authorization-logic/src/test/test_queries.rs

```diff
@@ -66,7 +66,8 @@ pub mod test {
 
     pub fn setup_dirs_and_run_query_test(t: QueryTest) {
         utils::setup_directories_for_bazeltest(vec![&t.input_dir], vec![&t.output_dir]);
-        run_query_test(t)
+        run_query_test(t);
+        clean_test_dir()
```

Comment:
@aferr was this the intent? It is actually dead code otherwise.

Reply:
Yes, that's the intent. I'm not sure why it wasn't like this already...